package.json

```diff
@@ -8,13 +8,7 @@
     "yarn": "^1.2.0"
   },
   "browserslist": [
-    ">0.2%",
-    "not dead",
-    "not ie <= 11",
-    "not op_mini all",
-    "not chrome < 51",
-    "not safari < 10",
-    "not android < 51"
+    "defaults"
```

Comment:
A choice, and it helps clean-up some build issues. We can discuss if this feels like an issue for any reason.

Reply:
I did at some point have to add  "not android < 51" to get may layers to work, I dont remember why... but lets test map layers here

Reply:
Gotcha. Have tested now and Map layers appear to be working well.  Tried with all possible layers on and a group shape also. Switched between and on and off, zoomed and navigated around and everything was happy. 

Also "defaults" is one of the suggested browserlist settings on the offical MapBox docs (with the addition of "not ie 11", but it turns out that is now covered in the current defaults as part of "dead"):

https://docs.mapbox.com/mapbox-gl-js/guides/install/#targeting-transpilation-to-es6-with-browserslist
https://browsersl.ist/#q=defaults%2C+not+ie+11 

So of course could still be somoething there, but I think we're good.